storagegroup/types.proto

```diff
@@ -11,6 +11,11 @@ import "refs/types.proto";
 // that require payed storage guaranties are gathered in `StorageGroups` with
 // additional information used for proof of storage. `StorageGroup` only
 // contains objects from the same container.
+//
+// StorageGroup may have expiration Epoch set with `__NEOFS__EXPIRATION_EPOCH`
+// well-known attribute. When expired, StorageGroup will be ignored by InnerRing
```

Comment:
I would clarify the association with the `STORAGE_GROUP` object somehow, for example, start the statement with `Being an object payload, ...`. Otherwise, the `well-known attribute` looks taken out of context IMO.